.github/CODEOWNERS

```diff
@@ -1,55 +1,102 @@
 # GitHub CODEOWNERS definition
 # See: https://help.github.com/articles/about-codeowners/
 
-# * @elastic/beats
+# The beats repository is owned by the @elastic/elastic-agent-data-plane Many teams contribute to it. The goal
+# is to cover all directories in the CODEOWNERS file. The list is sorted alphabetically by directory and sub directories.
 
-# libbeat
-/libbeat/ @elastic/elastic-agent-data-plane
-# /auditbeat/ @elastic/beats
-# /packetbeat/ @elastic/beats
-# /filebeat/ @elastic/beats
-# /metricbeat/ @elastic/beats
-# /journalbeat/ @elastic/beats
-# /winlogbeat/ @elastic/beats
-
-# Auditbeat
-/auditbeat/module/ @elastic/security-external-integrations
+* @elastic/elastic-agent-data-plane
+
+
+### Private directories
+# Any changes to the CI system or .github must be reviewed by the data plane team.
+/.ci/ @elastic/elastic-agent-data-plane
+/.github/ @elastic/elastic-agent-data-plane
+
+### Auditbeat
+/auditbeat/ @elastic/security-external-integrations
 /x-pack/auditbeat/ @elastic/security-external-integrations
 
-# Packetbeat
-/packetbeat/protos/ @elastic/security-external-integrations
-/x-pack/packetbeat/ @elastic/security-external-integrations
+### Deploy
+# All changes to the different deployments must be reviewed by data plane team.
+# TODO: This where k8s falls
```

Comment:
Does the data plan team have to review all changes to k8s for example?

Reply:
I do not think so.

Reply:
I don't mind the data plane team being listed as an owner here. SDHs for filebeat/metricbeat on k8s will probably end up with the data plane team, we should know what the deployment set up is.

We could list two codeowners here, one for the beat and one for whoever maintains the deployment types.

Reply:
@cmacknz Do you mean for `/deploy/kubernetes` we add data plane team and the team from @ChrsMark ? I like this idea because it ensures any change to k8s has the eyes from both teams on it.

Reply:
Yes, let's add both teams.